### DISCUSSION TITLE
🎉 Hacktoberfest 2024 is here! 🎉

### DISCUSSION THREAD

**Discussion by mauromorales** [kairos-io/kairos#2924] [Announcements]
Community, we have some swag for the first 10 contributors to Hacktoberfest in any of our repos. 

- The PR needs to be accepted by the team, and it cannot be a trivial change like a typo.
- FIFO (sorry, we're just limited with the amount of swag and costs to send. If there's a chance, we'll increase the number of winners but cannot promise)
- PR needs to be done during Hacktoberfest (October 2024)

We'll track the winners here, so if you don't see your name but think you should be there, just add a comment with a link to the PR and ping me @mauromorales 

0. https://github.com/kairos-io/packages/pull/1081 by @bencorrado (Plus reported and discussed a couple of other issues as well!)
1. https://github.com/kairos-io/kairos-docs/pull/290 by @6ixfalls
2. https://github.com/kairos-io/kairos-docs/pull/298 & https://github.com/kairos-io/kairos-docs/pull/296 by @sarthak-kumar-shailendra

**Comment by bencorrado** [comment]
Thanks!  Glad to be participating with you fine folks!

**Comment by mauromorales** [comment]
@bencorrado @6ixfalls @sarthak-kumar-shailendra please send me your t-shirt size and your address. You can find different ways to contact me on my user profile

**Comment by sarthak-kumar-shailendra** [comment]
Thanks! Glad to contribute.
